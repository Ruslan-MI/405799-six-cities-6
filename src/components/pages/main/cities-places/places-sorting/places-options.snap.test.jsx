import React from "react";
import {
  render,
} from "@testing-library/react";
import PlacesOptions from "./places-options";
import {
  StoreNameSpace,
} from "../../../../../const";
import Test, {
  mockMainPageInitialState,
} from "../../../../../mocks-for-test";

it(`Places options should render correctly`, () => {
  const onClick = jest.fn();

  const dataForStore = {
    [StoreNameSpace.MAIN_PAGE]: mockMainPageInitialState,
  };

  const {
    container,
  } = render(<Test dataForStore={dataForStore}>
    <PlacesOptions isPlacesOptionsOpened onClick={onClick} />
  </Test>);

  expect(container).toMatchSnapshot();
});
