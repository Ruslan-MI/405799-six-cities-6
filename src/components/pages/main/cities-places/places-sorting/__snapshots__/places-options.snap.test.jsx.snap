// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places options should render correctly 1`] = `
<div>
  <ul
    class="places__options places__options--custom places__options--opened"
  >
    <li
      class="places__option places__option--active"
      tabindex="0"
    >
      Popular
    </li>
    <li
      class="places__option "
      tabindex="0"
    >
      Price: low to high
    </li>
    <li
      class="places__option "
      tabindex="0"
    >
      Price: high to low
    </li>
    <li
      class="places__option "
      tabindex="0"
    >
      Top rated first
    </li>
  </ul>
</div>
`;
